dataland-frontend/src/frameworks/lksg/ViewConfig.ts

```diff
@@ -97,19 +97,20 @@ export const lksgViewConfiguration: MLDTConfig<LksgData> = [
             explanation: 'What share of the total number of employees in your company is made up by temporary workers?',
             shouldDisplay: (dataset: LksgData): boolean =>
               dataset.general?.masterData?.seasonalOrMigrantWorkers == 'Yes',
-            valueGetter: (dataset: LksgData): AvailableMLDTDisplayObjectTypes => {
-              const mappings = {
-                Smaller10: '<10%',
-                Between10And25: '10-25%',
-                Between25And50: '25-50%',
-                Greater50: '>50%',
-              };
-              return formatStringForDatatable(
-                dataset.general?.masterData?.shareOfTemporaryWorkers
-                  ? getOriginalNameFromTechnicalName(dataset.general?.masterData?.shareOfTemporaryWorkers, mappings)
-                  : ''
-              );
-            },
+            valueGetter: (dataset: LksgData): AvailableMLDTDisplayObjectTypes =>
+              ((): AvailableMLDTDisplayObjectTypes => {
+                const mappings = {
+                  Smaller10: '<10%',
+                  Between10And25: '10-25%',
+                  Between25And50: '25-50%',
+                  Greater50: '>50%',
```

Comment:
not your issue but: what do i choose in case of 25%?

Reply:
Is marked as a change because I did not merge main into branch earlier